core/src/main/java/com/graphhopper/storage/MMapDataAccess.java

```diff
@@ -55,6 +53,9 @@
  */
 public final class MMapDataAccess extends AbstractDataAccess {
 
+    private static final Logger LOGGER = LoggerFactory.getLogger(MMapDataAccess.class);
+    private static final MethodHandle BYTE_BUFFER_CLEANER = getByteBufferCleaner();
```

Comment:
We can also switch to an anonymous class implementing `Supplier<MethodHandle>` if we want the exception on each call.